catalog/transformer/catalog-transformer-video-input/src/main/resources/metadata.xslt

```diff
@@ -0,0 +1,26 @@
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
```

Comment:
Is metadata.xslt the same in both the tika and video input transformer projects?  If so, can we move this to a common area to avoid duplication?